src/main/java/no/nav/bilag/auth/TokenResponse.java

```diff
@@ -0,0 +1,10 @@
+package no.nav.bilag.auth;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Trenger angivelig ikke denne, ref https://nav-it.slack.com/archives/C6Y1W1HU7/p1693905680154119?thread_ts=1693904932.224019&cid=C6Y1W1HU7

Reply:
Nice price